src/contexts/NotificationProvider.tsx

```diff
@@ -24,14 +27,18 @@ export type NotificationOptions = {
 const NotificationContext = React.createContext({
   notifications: [],
   addNotification: (
+    // eslint-disable-next-line @typescript-eslint/no-unused-vars
     variant: NotificationVariantType,
+    // eslint-disable-next-line @typescript-eslint/no-unused-vars
     message: string,
+    // eslint-disable-next-line @typescript-eslint/no-unused-vars
     options?: NotificationOptions
+    // eslint-disable-next-line @typescript-eslint/no-unused-vars
   ) => null,
+  // eslint-disable-next-line @typescript-eslint/no-unused-vars
   removeNotification: (key: string) => null
 });
```

Comment:
Question: Could you define this as a function that doesn't take in any arguments? I know javascript is cool with passing in extra arguments, that was we wouldn't need all the eslint comments.

Not sure if typescript will get mad though.
```suggestion
  addNotification: () => null,
  // eslint-disable-next-line @typescript-eslint/no-unused-vars
  removeNotification: (key: string) => null
});
```

Reply:
Another potential option is to add an `_` in front of them - that generally denotes a var as unused but it may not work in this context.